pyaedt/edb_core/components.py

```diff
@@ -998,7 +998,7 @@ def add_port_on_rlc_component(self, component=None):
             )
             pin_layers = self._padstack._get_pin_layer_range(pins[0])
             pos_pin_term = self._pedb.edb.Cell.Terminal.PointTerminal.Create(
-                self._active_layout, pins[0].GetNet(), pins[0].GetName(), pt, pin_layers[0]
+                self._active_layout, pins[0].GetNet(), f"{component.refdes}_{pins[0].GetName()}", pt, pin_layers[0]
```

Comment:
@svandenb-dev The `f` string format is not working with IronPython.
You must use the old: `.format()`
```suggestion
                self._active_layout, pins[0].GetNet(), "{0}_{1}".format(component.refdes, pins[0].GetName()), pt, pin_layers[0]
```